review_practice/react-app/src/containers/SearchBar/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import Style from './style';
+
+import SearchButton from './SearchButton';
+import Logo from 'components/Logo';
+import Modal from 'components/Modal';
+import { DateFilter, PersonnelFilter, PriceFilter } from './ModalBody';
+
+import { isMobile } from 'Constants';
+import { clearToken } from 'service/TokenManager';
+
+export default ({ history }) => {
+    const dateButtonState = useState({ isClicked: false, buttonName: '날짜' });
+    const personnelButtonState = useState({ isClicked: false, buttonName: '인원' });
+    const priceButtonState = useState({ isClicked: false, buttonName: '가격' });
+    const buttonStateList = [dateButtonState, personnelButtonState, priceButtonState];
+    const buttonModalBodyList = [
+        (buttonState, setButtonState) => <DateFilter buttonState={buttonState} setButtonState={setButtonState} />,
+        (buttonState, setButtonState) => <PersonnelFilter buttonState={buttonState} setButtonState={setButtonState} />,
+        (buttonState, setButtonState) => <PriceFilter buttonState={buttonState} setButtonState={setButtonState} />,
+    ];
+
+    const searchStates = buttonStateList.map((state, index) => {
```

Comment:
```suggestion
    const searchStates = buttonStateList.map(([state setState], index) => {
```

state에 대한 배열에 대한 뜻을 알지 못한다면 이해하기 어려울 것 같습니다. 명시적으로 이름을 지정해주시면 좋을 것 같습니다.